### PR TITLE
test: Add additional testing

### PR DESCRIPTION
**Pull request by s2an**
Added test and small edits

### CODE REVIEW

**Review by n0rdie — COMMENTED** [review]
Passes so its good

File: lib/team.rb [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
               :abbreviation,
               :stadium,
               :link,
               :team_list
   
   def initialize(row, team_list)
     @team_id       = row[:team_id]
-    @franchise_id  = row[:franchise_id]
-    @team_name     = row[:team_name]
+    @franchiseid   = row[:franchiseid]
```
Comment (megroth2) [inline lib/team.rb:12]: Too late to make any adjustments, buuut I think technically following proper ruby convention would be:
```
@franchise_id = row[:franchiseid]
@team_name = row[:teamname]
```
I don't think we have to carry over the naming convention of the file headers into our application. We still want our attributes to follow snake case.

Comment (s2an) [reply lib/team.rb:12]: Good eye! That was an error on my part, I just wanted to guarantee it working :)

Comment (megroth2) [reply lib/team.rb:12]: Yeah no problem! Working code is more important than convention, so glad we at least got that fixed. If we'd had a tad more time we could've cleaned it up.

**Pull request merged**